.github/workflows/test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       # Must match `shard` definition in the test matrix:
       - name: Run tests
-        run: PYTHONPATH='.' pytest --num-shards=4 --shard-id=${{ matrix.shard }}
+        run: PYTHONPATH='.' pytest --num-shards=4 --shard-id=${{ matrix.shard }} tests
```

Comment:
It appears `django_stubs_ext/tests` ran twice previously. Once under `test` job and once under `matrix-test` job